pandas/tests/io/parser/test_encoding.py

```diff
@@ -252,6 +252,26 @@ def test_encoding_memory_map(all_parsers, encoding):
     tm.assert_frame_equal(df, expected)
 
 
+@skip_pyarrow
+def test_chunk_splits_multibyte_char(all_parsers):
+    """
+    Chunk splits a multibyte character with memory_map=True
+
+    GH 43540
+    """
+    parser = all_parsers
+    # DEFAULT_CHUNKSIZE = 262144, defined in parsers.pyx
+    df = DataFrame(data=["a" * 127] * 2048)
```

Comment:
If you want to, you could replace "a" with only "ą"s - in case the default chuncksize changes at some point.

Reply:
I'd rather keep it as it is. If `chunksize` grows significantly, replacement of single byte encoding with two bytes will not help much. I wonder if there's a way to import the value of `chunksize` into Python code; as far as I know it's defined only in the parser's C code.

Reply:
sure you can import it to python code, make sure is just defined w/o a cdef (careful you have to actually define it this way in *cython* to make it performant, but you can have *another* variable that's exportable)

Reply:
Since this fixes a regression, I'm happy with keeping the changes minimal. Keeping it as-is, is fine with me.